src/baseline_bots/utils.py

```diff
@@ -299,6 +300,36 @@ def get_state_value(bot, game, power_name):
     return len(game.get_centers(power_name))
 
 
+@gen.coroutine
+def get_best_orders(bot: BaselineBot, proposal_order: dict, shared_order: dict):
+    """
+    input: sender power, dipnet_order + incoming proposals {power: [orders]}, shared_orders, Diplomacy game
+    output: [orders] a list of orders (with best value)
+                # for each xdo order set (max at 6 for now) -> simulate worlds by execute all of shared orders + xdo order set
```

Comment:
Why is there a # comment inside the docstring?

Reply:
oops I forgot to remove